Susi/Controllers/LoginController/LoginVCMethods.swift

```diff
@@ -60,9 +60,16 @@ extension LoginViewController {
             self.skipButton.isEnabled = value
             self.signUpButton.isEnabled = value
             if value {
-                self.alert.dismiss(animated: true, completion: nil)
+                self.countValue += 1
```

Comment:
Do you think it is best idea to use count for alert? It will show for only one time. There is Observers, use that for continuous getting internet status.

Reply:
@jogendra , actually i used count for overcoming the situation of first time notification .when the functions whenReachable and whenUnreachable  are called the value becomes true and we counted the value with the help of count . 
The alert is showing only single time because of the simulator, as when we test these functions in simulator , it called only for a single time , but it will behave perfectly in physical device 
Please let me know that you tested in simulator or on physical device? , If you tested in physical device and it is showing the alert for the single time ,then definitely , there is a problem and i will fix it using a new mechanism for reachability.

Reply:
@baquer I testes on simulator only, I will test on a physical device and let you know.